.circleci/config.yml

```diff
@@ -337,13 +390,16 @@ workflows:
           requires:
             - ios-hold-testflight
 
-      - android-build:
+      - android-play-build:
+          requires:
+            - lint-testunit
+      - android-foss-build:
```

Comment:
IMO we should only trigger foss build in releases.
Otherwise we might be spending too many machines.